armi/utils/plotting.py

```diff
@@ -404,13 +404,12 @@ def plotFaceMap(
             )
     elif referencesToKeep:
         # Don't show yet, since it will be updated.
+        plt.close(fig)
```

Comment:
please Delete this one. it cant close with `plotDepthMap`

Reply:
ope! @john-science I guess we didn't test this internally